jwql/utils/credentials.py

```diff
@@ -23,6 +23,40 @@
 from jwql.utils.utils import get_config, check_config_for_key
 
 
+def get_mast_base_url(request=None):
+    """Return base url for mnemonic query.
+
+    Parameters
+    ----------
+    request : HttpRequest object
+        Incoming request from the webpage
+
+    Returns
+    -------
+    token : str or None
+        Base url for MAST JWST EDB API.
+    """
+    if Mast.authenticated():
+        print('Authenticated with Astroquery MAST magic')
+        return None
+    else:
```

Comment:
It appears that these lines are causing the second panel of the EDB page to break on the dev, test, and ops web servers.  On those servers, this `if` statement passes, and so the function returns `None`, which causes the `base_url` in `engineering_database.py` to be the default one, which doesn't currently work.  Thus, we need this function to return the `base_url` provided either in the `config.json` file or the environment variable.

As such, I think the only changes necessary here are to simply remove these lines that I've highlighted (and properly indent the rest of the function).

Reply:
Went ahead and removed the logic. We are getting some test failures. The last run failure was because of a timeout. I Just kicked them off again so we will see what happens 🤞